index.html

```diff
@@ -1,16 +1,121 @@
 <!DOCTYPE html>
 <html lang="en">
```

Comment:
All well done in a fast and precise manner! You have shown use of responsive elements and completed more than required and deployed the page to a real website. Great work, keep it going!